mapviz/include/mapviz/mapviz_plugin.h

```diff
@@ -220,6 +220,7 @@ namespace mapviz
 
     virtual void Transform() = 0;
 
+    virtual void UpdateConfig(std::map<std::string, std::string>& params) {}
```

Comment:
Could we modify to swri_yaml_util to add a function to parse a YAML string?  You could easily pack the key value pairs in an AddMapvizDisplay request into a simple YAML dictionary, parse it, and then pass it to the normal LoadConfig instead of having to add a new method.